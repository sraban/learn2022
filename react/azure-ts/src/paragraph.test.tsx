import React from "react";
import { shallow } from "enzyme";
import Paragraph from "./paragraph";

describe("Paragraph", () => {
    it('should render passed in string in a paragraph', () => {
        const paragraph = shallow(<Paragraph passedInText="Azure DevOps and SonarQube Rocks!" />);
        expect(paragraph).toMatchSnapshot();
    });
});