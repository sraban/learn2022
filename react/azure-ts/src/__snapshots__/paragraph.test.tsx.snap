// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph should render passed in string in a paragraph 1`] = `
<div
  className="container"
>
  <p>
    Azure DevOps and SonarQube Rocks!
  </p>
</div>
`;
